static/js/reducers/email.js

```diff
@@ -3,18 +3,33 @@ import {
   START_EMAIL_EDIT,
   UPDATE_EMAIL_EDIT,
   CLEAR_EMAIL_EDIT,
+  UPDATE_EMAIL_VALIDATION,
 } from '../actions/email';
 import type { Action } from '../flow/reduxTypes';
 
-export type EmailEditState = {
-  subject?:   ?string;
-  body?:      ?string;
-  query?:     ?Object;
+export type Email = {
+    subject?:   ?string;
+    body?:      ?string;
+    query?:     ?Object;
+};
+
+export type EmailValidationErrors = {
+    subject?:   ?string;
+    body?:      ?string;
+    query?:     ?string;
```

Comment:
As of now there isn't a way to, I had thought we might want to use this for reporting if the user is trying to send an email to 0 users.